src/Bridge/Symfony/DependencyInjection/FidryAliceDataFixturesExtension.php

```diff
@@ -44,12 +41,12 @@ public function load(array $configs, ContainerBuilder $container)
         $processedConfiguration = $this->processConfiguration($configuration, $configs);
         $bundles = array_flip($container->getParameter('kernel.bundles'));
 
-        if (false === array_key_exists(self::NELMIO_ALICE_BUNDLE_CLASS, $bundles)) {
+        if (false === array_key_exists(NelmioAliceBundle::class, $bundles)) {
```

Comment:
The check is the same, as the FQCN resolution happens at compile time.
So if the class does not exists it doesn't break.

However, this modification was pushed by error, as it wasn't intended to be in this PR, I'll revert it if you prefer.